templates/support/index.html

```diff
@@ -139,578 +171,747 @@ <h2 class="p-heading--4">Security, support and managed open source for enterpris
-            width="250",
-            height="56",
-            hi_def=True,
-            loading="auto",
-        ) | safe
-      }}
-    </div>
-    <div class="col-7">
-      <h2>Learn how ITstrategen keeps their applications secure with Ubuntu</h2>
-      <p>The security of customer data is of the utmost importance to ITstrategen, which is why Ubuntu is their server operating system of choice.</p>
-      <a class="p-button" href="/engage/ITstrategen-case-study">Read the case study</a>
+    <div class="col">
+      <p>Canonical offers Ubuntu Pro + Support at a discount for approved institutions, such as schools, research and academia.</p>
+      <p>
+        <a class="p-button" href="/support/contact-us?product=education-discount">Contact us for pricing</a>
+      </p>
     </div>
   </div>
 </section>
 
-<section id="community-support" class="p-strip--light is-deep">
-  <div class="row u-equal-height">
-    <div class="col-8">
-      <h2>From VMware to Open Source</h2>
-        
-      <p>Are you looking for VMware alternatives? Think open source: the world's leading software portfolio. Open-source software enables you to build fully functional cloud infrastructure while ensuring TCO reduction and innovation advances.</p>
-      <p><a class="p-button--positive" href="/support/community-support">Watch the webinar</a></p>
+<section class="p-section--deep">
+  <hr class="p-rule is-fixed-width">
+  <div class="row--50-50">
+    <div class="col">
+      <h2>Learn more</h2>
     </div>
-    <div class="col-4 u-hide--medium u-hide--small u-align--center u-vertically-center">
-      {{
-        image(
-            url="https://assets.ubuntu.com/v1/168111aa-Cloud+migration.svg",
-            alt="",
-            width="150",
-            height="150",
-            hi_def=True,
-            loading="lazy"
-        ) | safe
-      }}
+    <div class="col">
+      <ul class="p-list--divided">
+        <li class="p-list__item"><a href="/training">Get training</a></li>
+        <li class="p-list__item"><a href="/support/contact-us?product=support-overview">Contact us</a></li>
+        <li class="p-list__item"><a href="/pro/subscribe">Get support now</a></li>
+      </ul>
     </div>
   </div>
 </section>
 
-{% with first_item="_support_landscape", second_item="_support_contact_us", third_item="_further_reading" %}{% include "shared/contextual_footers/_contextual_footer.html" %}{% endwith %}
-
 <!-- Set default Marketo information for contact form below-->
-<div class="u-hide" id="contact-form-container" data-form-location="/shared/forms/interactive/support" data-form-id="1240" data-lp-id="2065" data-return-url="https://ubuntu.com/support/thank-you" data-lp-url="https://pages.ubuntu.com/things-contact-us.html">
+<div class="u-hide" id="contact-form-container" data-form-location="/shared/forms/interactive/support" data-form-id="1240" data-lp-id="2065" data-return-url="/support/thank-you" data-lp-url="https://pages.ubuntu.com/things-contact-us.html">
```

Comment:
I'm not sure, but I assume the link should stay absolute as before, since this might be passed to BE to return to FE. So relative path might not work.

Reply:
Good catch. Yes, it doesn't work because marketo doesn't know what's the original host. I removed it accidentally when I did the global replacement in the file.